dotcom-rendering/src/components/FlexibleGeneral.tsx

```diff
@@ -217,6 +221,7 @@ const decideCardProperties = (
 				headlineSizeOnMobile: 'large',
 				headlineSizeOnTablet: 'large',
 				imageSize: 'medium',
+				useBoostedFontSizes: false,
```

Comment:
If possible, we should try and stick to the boosted font sizes when a card is boosted to stop the logic getting messy. If necessary we can extend the types to account for extra sizes. Probably we should seperate the default and boost case here too if they require different sizes.